easybuild/easyconfigs/e/EveryBeam/EveryBeam-0.5.2-foss-2023b.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'CMakeMake'
+
+name = 'EveryBeam'
+version = '0.5.2'
+
+homepage = 'https://everybeam.readthedocs.io/'
+description = """Library that provides the antenna response pattern for several instruments,
+such as LOFAR (and LOBES), SKA (OSKAR), MWA, JVLA, etc."""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+
+sources = [
+    {
+        'filename': '%(name)s-v%(version)s.tar.gz',
+        'git_config': {
+            'url': 'https://git.astron.nl/RD',
+            'repo_name': '%(name)s',
+            'tag': 'v%(version)s',
+            'clone_into': '%(name)s',
+            'recursive': True
+        }
+    },
+]
+checksums = [None]
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+    ('wget', '1.21.4'),
+]
+dependencies = [
+    ('casacore', '3.5.0'),
+    ('Boost', '1.83.0'),
+    ('CFITSIO', '4.3.1'),
+    ('WCSLIB', '7.11'),
+    ('GSL', '2.7'),
+    ('HDF5', '1.14.3'),
+    ('Python', '3.11.5'),
+    ('libxml2', '2.11.5'),
+]
+
+# Without this, the wrong (system) python is picked up by cmake
+# in cases where the system python is newer than the one used in these EasyConfigs
+# See PR # 19119
+configopts = "-DPython3_EXECUTABLE=$EBROOTPYTHON/bin/python "
```

Comment:
This should now be fixed in the CMakeMake EasyBlock by https://github.com/easybuilders/easybuild-easyblocks/pull/3088 . Please remove this configopts addition and check it still works (we can check when building for EESSI, where you found the issue previously....)